apps/console/src/features/userstores/constants/user-store-constants.ts

```diff
@@ -90,3 +90,11 @@ export const USERSTORE_REGEX_PROPERTIES = {
  * @type {string}
  */
 export const DISABLED = "Disabled";
+
+/**
+ * Primary userstore name.
+ *
+ * @constant
+ * @type {string}
+ */
+export const PRIMARY_USERSTORE = "PRIMARY";
```

Comment:
This constant is already available in core. Better to use that.

https://github.com/wso2/identity-apps/blob/master/modules/core/src/constants/is-constants.ts#L39

Reply:
Shouldn't that come under https://github.com/wso2/identity-apps/blob/master/modules/core/src/constants/userstore-constants.ts?